src/course-page-resources/course-page/CoursePageConfigCard.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@edx/paragon';
+import classNames from 'classnames';
+
+import messages from '../messages';
+
+function CoursePageConfigCard({
+  intl,
+  id,
+  title,
+  description,
+  pageStatus,
+  showSettings,
+}) {
+  const cardSettings = 'bg-white shadow p-3 m-0 my-3 course-page-title';
+  const cardClasses = classNames('card', cardSettings, { disabled: !pageStatus });
+  const cardSubSettings = 'bg-white px-3 py-0';
+
+  const buttonSettings = 'd-flex mt-0 justify-content-center align-items-center';
+  const buttonClasses = classNames(buttonSettings, { visible: !pageStatus, invisible: pageStatus });
+  const pageStatusMsgId = pageStatus ? 'pageStatus.enabled' : 'pageStatus.disabled';
+
+  return (
+    <div className={cardClasses} key={id}>
+      <div className={classNames('card-header', cardSubSettings)}>
+        <div className="row">
+          <h2 className="font-weight-bold">{title}</h2>
+          <div className="col flex-grow" />
+          {showSettings && <FontAwesomeIcon icon={faCog} />}
+        </div>
+      </div>
+      <div className={classNames('card-body', cardSubSettings)}>
+        {showSettings && <p>{intl.formatMessage(messages[pageStatusMsgId])}</p>}
+        <p>{description}</p>
+      </div>
+      <div className={classNames('card-footer', cardSubSettings)}>
+        <div className={buttonClasses}>
+          {!showSettings && (
```

Comment:
Showing the enabled button has nothing to do with settings? Why is it linked here?

Reply:
It is not "enabled" (status) but "enable" (action), and it is shown only when the settings icon is not shown.

Reply:
As I mentioned in the previous comment, the mockup doesn't dictate this functonality.